estimator/transport_co2/model.py

```diff
@@ -28,16 +28,24 @@ def __init__(self, co2_per_vehicle_km: float, average_occupancy: float):
     BUS = (863, 12.7)
 
     def estimate_co2(
-        self, distance_in_km: float, occupancy: Optional[float] = None
+        self,
+        distance_in_km: float,
+        *,
+        co2_per_vehicle_km: Optional[float] = None,
+        occupancy: Optional[float] = None
     ) -> float:
         """
         Estimate CO2 usage for transport mode based on KM and optional vehicle occupancy.
 
         Keyword arguments:
-            distance_in_km -- distance for the trip in kilometers
-            occupancy -- optional vehicle occupancy (uses average occupancy if falsey)
+            distance_in_km -- distance for the trip in kilometers (required)
+            co2_per_vehicle_km -- CO2 emission in grams per vehicle km
+                (optional, uses mode average if falsey)
+            occupancy -- vehicle occupancy (optional, uses mode average if falsey)
         """
+
+        co2_per_vehicle_km = co2_per_vehicle_km or self.avg_co2_per_vehicle_km
         # occupancy should be above zero
```

Comment:
Should we explicitly check for this in the code? Are negative values falsy?

Reply:
No, only `0` (or `0.0`) is falsey. We could do an explicit check, but I generally wouldn't put this into the body, but rather the contract of the method, i.e., put the responsibility on the caller. Else we might be wasting a lot of CPU cycles doing unnecessary checks when the caller guarantees the condition is already fulfilled.